### PR TITLE
Feature/vudrm 1722/multi key cpix

### PR DESCRIPTION
**Pull request by PerritonBranch**

### CODE REVIEW

File: docs/source/DeveloperDocumentation/VUDRM-key-provision.md [hunk 1]
```diff
@@ -201,6 +221,221 @@ curl -XGET -H "API-KEY: <api-key>" \
 </cpix:CPIX>
 ```
 
+#### Multikey with seperate bitrates based key
+
+Get a CPIX 2.1 document that uses different content keys per bitrate and one for audio.
+If given a single bitrate is given the document will use one key for all bitrates up to the given bitrate, and one key for the given bitrate up.
```
Comment (bengo1990) [inline docs/source/DeveloperDocumentation/VUDRM-key-provision.md:227]: Quite a lot of givens - but the first given should be removed.

**Review by bengo1990 — CHANGES_REQUESTED** [review]
Just a text amend as per the comment.

File: docs/source/DeveloperDocumentation/VUDRM-key-provision.md (hunk 1 above)
Comment (PerritonBranch) [reply docs/source/DeveloperDocumentation/VUDRM-key-provision.md]: Yep thats my b, didn't re-write the sentence correctly haha

**Review by bengo1990 — DISMISSED** [review]

File: docs/source/DeveloperDocumentation/VUDRM-key-provision.md [hunk 2]
```diff
@@ -201,6 +221,220 @@ curl -XGET -H "API-KEY: <api-key>" \
 </cpix:CPIX>
 ```
 
+#### Multikey with seperate bitrates based key
+
+Get a CPIX 2.1 document that uses different content keys per bitrate and one for audio.
+If provided with a single bitrate the document will use one key for all bitrates up to the given bitrate, and one key for the given bitrate up.
+If provided with two bitrates the document will use one key for all bitrates up to the lowest given bitrate, one key for all bitrates between the lowest and highest bitrates, and one key for the highest given bitrate up.
+
+##### Quiry params:
```
Comment (somerobmitchell) [inline docs/source/DeveloperDocumentation/VUDRM-key-provision.md:230]: * query

File: docs/source/DeveloperDocumentation/VUDRM-key-provision.md [hunk 3]
```diff
@@ -201,6 +221,220 @@ curl -XGET -H "API-KEY: <api-key>" \
...
+    </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### Multikey with seperate video quality based keys
+
+Get a CPIX 2.1 document that uses different content keys per video qualty and one for audio.
+If provided with multiple video qualities the response will contain a encryption key for each video quality and one encrypt key for everything else.
+
+##### Quiry params:
```
Comment (somerobmitchell) [inline docs/source/DeveloperDocumentation/VUDRM-key-provision.md:326]: query

**Review by somerobmitchell — DISMISSED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Review by bengo1990 — APPROVED** [review]

**Pull request merged**